src/object/srv_obj_migrate.c

```diff
@@ -2579,7 +2562,7 @@ migrate_one_epoch_object(daos_epoch_range_t *epr, struct migrate_pool_tls *tls,
 	/* Only open with RW flag, reintegrating flag will be set, which is needed
```

Comment:
may need to update comment here "RW flag"
and why must change DAOS_OO_RW to DAOS_OO_RO for dsc_obj_list_obj()? to include the DOWN tgt in the layout? (will do degraded enumerate for the DOWN tgt?)

Reply:
Because read-only case will only fetch the data, so during reintegratin/rebuilding, the layout might include some rebuilding shard, which might be healthy shard, and also make sure the layout is calculated by the original pool map, so DOWN is allowed status here.

DOWN tgt will be skipped by client read stack.  This is mainly used by rebuild/migration.